blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpAggregatedMultiIngestionHandler.java

```diff
@@ -130,9 +129,6 @@ public void handle(ChannelHandlerContext ctx, HttpRequest request) {
             log.debug(String.format("BAD JSON: %s", body));
             log.error(ex.getMessage(), ex);
             DefaultHandler.sendResponse(ctx, request, ex.getMessage(), HttpResponseStatus.BAD_REQUEST);
-        } catch (ConnectionException ex) {
```

Comment:
Hmm... I guess the message is different. I want to remove references to Astyanax specific classes from this class. I will change the message for ```catch(Exception ex)``` to be more generic